gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -755,10 +786,40 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
        call ESMF_ConfigGetAttribute ( cfg, value=list(n)%mode,default='instantaneous', &
                                       label=trim(string) // 'mode:' ,rc=status )
        _VERIFY(STATUS)
-       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%descr, &
+
+       ! Fill the global attributes
+
+       ! filename is special as it does double duty, so we fill directly
+       ! from HistoryCollection object
+       list(n)%global_atts%filename = list(n)%filename
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%descr, &
                                       default=INTSTATE%expdsc, &
                                       label=trim(string) // 'descr:' ,rc=status )
        _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%comment, &
+                                      default=INTSTATE%global_atts%comment, &
+                                      label=trim(string) // 'comment:' ,_RC)
+       _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%contact, &
+                                      default=INTSTATE%global_atts%contact, &
+                                      label=trim(string) // 'contact:' ,_RC)
+       _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%convention, &
+                                      default=INTSTATE%global_atts%convention, &
+                                      label=trim(string) // 'convention:' ,_RC)
+       _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%institution, &
+                                      default=INTSTATE%global_atts%institution, &
+                                      label=trim(string) // 'institution:' ,_RC)
+       _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%references, &
+                                      default=INTSTATE%global_atts%references, &
+                                      label=trim(string) // 'references:' ,_RC)
+       _VERIFY(STATUS)
+       call ESMF_ConfigGetAttribute ( cfg, value=list(n)%global_atts%source, &
+                                      default=INTSTATE%global_atts%source, &
+                                      label=trim(string) // 'source:' ,_RC)
```

Comment:
Remove the `_VERIFY`s also.

Reply:
I feel dumb... :)

Reply:
Well ... given that it is something I would have done, I'm hoping for a less severe perjorative.